src/Apps/Settings/SettingsApp.tsx

```diff
@@ -51,9 +53,28 @@ const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
     <>
       <MetaTags title="Settings | Artsy" />
 
-      <Text variant={["lg-display", "xl"]} mt={4}>
-        Hi, {me.name}
-      </Text>
+      {isCollectorProfileEnabled ? (
+        <>
+          <Media greaterThan="xs">
+            <TopContextBar
+              displayBackArrow
+              useWithoutSeparator
+              href="/collector-profile" // TODO: use onClick and router or
```

Comment:
**thought:** do we have a ticket for this 👀

Reply:
Nope, it's just a reminder, as 'nice to have' most probaly